docs/csv/example_files/ballot_measure_contest.txt

```diff
@@ -1 +1 @@
-abbreviation,Contest,ballot_measure_selection_ids,ballot_sub_title,ballot_title,elecoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation,con_statement,effect_of_abstain,full_text,info_uri,passage_threshold,pro_statement,summary_text,type,other_type,id
+abbreviation,Contest,ballot_measure_selection_ids,ballot_sub_title,ballot_title,elecoral_district_id,electorate_specification,external_identifier_type,external_identifier_othertype,external_identifier_value,has_rotation,name,sequence_order,vote_variation,other_vote_variation,con_statement,effect_of_abstain,full_text,info_uri,passage_threshold,pro_statement,summary_text,type,other_type,id
```

Comment:
Oops, I wasn't quite clear. My request was that the id columns always be first. My thinking behind this is that it's easier for someone to search through a file by id if you can

grep ^${ID_VALUE}, file.csv

instead of

grep ,${ID_VALUE} file.csv

which could cause issues if the value you're looking for is in other columns in the file.